docs/getting_started/index.rst

```diff
@@ -0,0 +1,167 @@
+the service IP as it will be used for the ReplicationSource.
+
+.. code-block:: bash
+
+   kubectl get svc -n dest
+   NAME                                TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+   scribe-rsync-dest-database-sample   ClusterIP   10.107.249.72   <none>        22/TCP    18s
+
+Now it is time to deploy our database.
+
+.. code-block:: bash
+
+   kubectl create ns source
+   kubectl create -n source -f examples/source-database
+
+Verify the database is running.
+
+.. code-block:: bash
+
+   kubectl get pods -n source
+   NAME                    READY   STATUS    RESTARTS   AGE
+   mysql-8b9c5c8d8-24w6g   1/1     Running   0          17s
+
+Now it is time to create the ReplicationSource items. First, we need the ssh secret from the
+dest namespace.
+
+.. code-block:: bash
+
+   kubectl get secret -n dest scribe-rsync-dest-src-database-destination -o yaml | sed 's/namespace: dest/namespace: source/g' > /tmp/secret.yaml
+   kubectl create -f /tmp/secret.yaml
+
+Using the service object that relates to the ReplicationDestination that was recorded earlier. Modify
+*scribe_v1alpha1_replicationsource.yaml* replacing the value of the address and create the ReplicationSource object.
+
+.. code-block:: bash
+
+   sed -i 's/my.host.com/10.107.249.72/g' examples/scribe_v1alpha1_replicationsource.yaml
+   kubectl create -n source -f examples/scribe_v1alpha1_replicationsource.yaml
+
+To verify the replication has completed describe the Replication source.
+
+.. code-block:: bash
+
+   kubectl describe ReplicationSource -n source database-source
+
+From the output, the success of the replication can be seen by the following lines.
+
+.. code-block:: bash
+
+   Status:
+     Conditions:
+       Last Transition Time:  2020-12-03T16:07:35Z
+       Message:               Reconcile complete
+       Reason:                ReconcileComplete
+       Status:                True
+       Type:                  Reconciled
+     Last Sync Duration:      4.511334577s
+     Last Sync Time:          2020-12-03T16:09:04Z
+     Next Sync Time:          2020-12-03T16:12:00Z
+
+Create a databases in the mysql pod running in the source namespace.
+
+.. code-block:: bash
+
+   kubectl exec --stdin --tty -n source `kubectl get pods -n source | grep mysql | awk '{print $1}'` /bin/bash
+   mysql -u root -p$MYSQL_ROOT_PASSWORD
+   show databases;
+   +--------------------+
+   | Database           |
+   +--------------------+
+   | information_schema |
+   | mysql              |
+   | performance_schema |
+   | sys                |
+   +--------------------+
+   4 rows in set (0.00 sec)
+
+
+   create database synced;
+   exit
+   exit
+
+Now the mysql database will be deployed to the dest namespace which will use the data that has been replicated.
+First list the snapshots, record the values of the latest snapshot as it will be used to create a pvc. Then create
```

Comment:
Also want to direct people to the destination `.status.latestImage` instead of listing VolumeSnapshots